gensim/models/keyedvectors.py

```diff
@@ -272,6 +273,15 @@ def word_vec(self, word, use_norm=False):
         else:
             raise KeyError("word '%s' not in vocabulary" % word)
 
+    def get_words_from_vocab(self):
```

Comment:
Oh, yes! Will change! I'm still keeping the get_words_from_vocab() because when I started out with gensim, it was confusing for me initially what the w2v.wv.vocab keys and items were for!
```python
    def get_words_from_vocab(self):
        """
        returns the words in the current vocabulary as a list.
        """

        if(not self.index2word):
            raise ValueError("Vocabulary needs to be built before calling this function")
        return self.index2word
```